src/components/Image.vue

```diff
@@ -25,7 +29,7 @@ export default defineComponent({
   data() {
     return {
       resourceUrl: '',
-      imageUrl: require("@/assets/images/defaultImage.png")
```

Comment:
I think we haven't handled the case for default image.

Reply:
Handled the case if we do not get the correct image.
Thankyou Sir, for reviewing my code.